Sources_v3/Query/UserListQuery.swift

```diff
@@ -51,3 +51,12 @@ public struct UserListQuery: Encodable {
         try options.encode(to: encoder)
     }
 }
+
+extension UserListQuery {
+    /// Builds `UserListQuery` for a user with the provided `userId`
+    /// - Parameter userId: The user identifier
+    /// - Returns: `UserListQuery` for a specific user
+    static func user(withID userId: UserId) -> Self {
+        .init(filter: .equal("id", to: userId))
+    }
```

Comment:
👀 We don't have a classic REST `users/<id>` GET call?

Reply:
Yep, there is no such